### DISCUSSION TITLE
Notifications not working

### DISCUSSION THREAD

**Discussion by AdamM68** [mcguirepr89/BirdNET-Pi#789] [Q&A] [answered]
I have a problem with Notifications.
When I put a config entry like (xxx.yyy  is an example ... I put there my name and code from google )
mailto://xxx:yyy@gmail.com
 and tick all notify and send weekly report and then ...
Send test notification - works ..it is sending an email :)
but when I press update settings , after a while it is showing Success ...but the config entry in Notifications section is empty and notifications are not sent  :(

It looks like "update" is not writing down my entry 

Any idea ?

**Comment by lloydbayley** [comment]
That is VERY odd. If you go into terminal (either Web Terminal in the Settings or a shell) have a look in the apprise.txt file to see if your entry is in there.

```cat $HOME/BirdNET-Pi/apprise.txt```

It should contain the line you put in including your username and password.
If it is blank, then there's something larger afoot!

**Comment by paulhoff** [comment]
I am having the exact same problem with Adam.  Trying to use Prowl.  I put all the info in, it will send a test message, but when I update everything in the notification section disappears.  It looks like the apprise.txt file is empty.

**Comment by lloydbayley** [comment, marked as answer]
Some changes were made that have been reversed. See here: #793 
An update should cure this.
Tools -> System Controls -> Update
